### ISSUE TITLE
Maintenant?

### ISSUE THREAD

**Issue by GuillouTran** [tmcw/bookish-api#47]
Are you still maintaining it by chance?

**Comment by tmcw** [comment]
Yes, I use https://bookish.tech/ for myself and update it when necessary. In terms of generating YAML metadata for my book reviews, this solution pretty much works. I'm happy to review relevant PRs.

**Comment by GuillouTran** [comment]
I see. But it’s hard to reproduce as the fact that Nextjs no longer maintains “micro”
